vehicle/mkvi/software/sensing/sensing.yml

```diff
@@ -0,0 +1,19 @@
+name: brakelight_bspd
+# subscribe:
+#   - name: air_control_critical
+#     mob: 1
```

Comment:
Does the sensing board need to listen to any CAN messages?

Reply:
I don't believe so! It's just reporting what it gets from wheel speed sensors

Reply:
What's consuming this data? Dashboard to send over telemetry?

Reply:
Data is just being logged for post-drive analysis.